spec/unit/uimanager_bench.lua

```diff
@@ -29,7 +29,7 @@ describe("UIManager checkTasks benchmark", function()
     for i=1, NB_TESTS do
         table.insert(
             UIManager._task_queue,
-            { time = now + i, action = noop, argc = 0, args = {} }
+            { time = now + i, action = noop, args = {} }
```

Comment:
Harmless here because the table is empty, but, FWIW, if you *do* need to set args, this *needs* to happen at table construction, and in *this* oder:

`{n = 4, 1, nil, nil, 4}`

(Leaving this here for future archeologists ;)).